src/components/layout/Footer.tsx

```
@@ -7,7 +7,7 @@ export default function Footer() {
                 {/* 상단 */}
                 <ul className="flex flex-wrap gap-x-4 gap-y-2 text-sm font-caption text-default-gray-800">
                     <li>
-                        <Link to="/">개인정보 처리방침</Link>
+                        <div onClick={() => alert('아직 지원하지 않는 기능이에요')}>개인정보 처리방침</div>
                     </li>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)